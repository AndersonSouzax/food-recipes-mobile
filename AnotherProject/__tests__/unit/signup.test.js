import 'react-native';
import React from 'react';

import SignUp from '../../src/signup';

jest.mock('../../src/services/api');

import { create, act } from 'react-test-renderer';

test('renders correctly', () => {
  const component = create(<SignUp />).toJSON();
  expect(component).toMatchSnapshot();
});




