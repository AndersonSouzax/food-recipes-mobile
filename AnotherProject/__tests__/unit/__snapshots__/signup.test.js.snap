// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#404040",
        "fontFamily": "cursive",
        "fontSize": 32,
        "fontWeight": "bold",
        "marginBottom": 25,
      }
    }
  >
     Food Recipes 
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    onKeyPress={[Function]}
    placeholder="email"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#EA1D2C",
        "borderRadius": 5,
        "borderWidth": 1,
        "fontSize": 16,
        "marginBottom": 15,
        "marginHorizontal": 20,
        "paddingHorizontal": 20,
        "paddingVertical": 15,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    onKeyPress={[Function]}
    placeholder="password"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#EA1D2C",
        "borderRadius": 5,
        "borderWidth": 1,
        "fontSize": 16,
        "marginBottom": 15,
        "marginHorizontal": 20,
        "paddingHorizontal": 20,
        "paddingVertical": 15,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    onKeyPress={[Function]}
    placeholder="repeat password"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderColor": "#EA1D2C",
        "borderRadius": 5,
        "borderWidth": 1,
        "fontSize": 16,
        "marginBottom": 15,
        "marginHorizontal": 20,
        "paddingHorizontal": 20,
        "paddingVertical": 15,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessible={true}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#EA1D2C",
        "borderRadius": 5,
        "margin": 15,
        "marginHorizontal": 10,
        "padding": 20,
      }
    }
    testID="signUpButton"
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontSize": 16,
          "fontWeight": "bold",
          "textAlign": "center",
        }
      }
    >
      Sign Up
    </Text>
  </View>
</View>
`;
